.github/workflows/pipeline.develop.yml

```diff
@@ -44,15 +44,17 @@ jobs:
       - name: JS Lint
         run: npm run lint:ts
       - name: Run coders examples
-        env:
-          BLOCKCHAIN_PROVIDER_URL: ${{ secrets.BLOCKCHAIN_PROVIDER_URL_DEVELOP }}
         run: npx hardhat run scripts/coders.ts
+      
+      # BSC
```

Comment:
do we need comment if we have name?

Reply:
I was talking with @dougmrqs about it. It is kind of hard to find a specific chain and step that we want to modify when needed. It is an idea to keep more organized this file.

Reply:
I just added a nl and removed the comment, what do you think?